blogs/dns_tracking/dns_tracking.md

```diff
@@ -61,13 +59,67 @@ collector object with the following fields enabled in eBPF config
 section
 
 ```yaml
+apiVersion: flows.netobserv.io/v1beta1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
   agent:
     type: EBPF
     ebpf:
       privileged: true
-      enableDNSTracking: true
```

Comment:
That should end like this:
https://github.com/netobserv/network-observability-operator/pull/394/files#r1304927133

Let's keep an eye on the final output